plugins/discourse-calendar/assets/javascripts/discourse/components/modal/post-event-builder.gjs

```diff
@@ -232,6 +241,50 @@ export default class PostEventBuilder extends Component {
       : null;
   }
 
+  // Native input handlers
+  @action
+  setName(e) {
+    this.event.name = e.target.value;
+  }
+
+  @action
+  setLocation(e) {
+    this.event.location = e.target.value;
+  }
+
+  @action
+  setUrl(e) {
+    this.event.url = e.target.value;
+  }
+
+  @action
+  setDescription(e) {
+    this.event.description = e.target.value;
+  }
+
+  @action
+  setShowLocalTime(e) {
+    this.event.showLocalTime = e.target.checked;
+  }
+
+  @action
+  setMinimal(e) {
+    this.event.minimal = e.target.checked;
+  }
+
+  @action
+  setChatEnabled(e) {
+    this.event.chatEnabled = e.target.checked;
+  }
```

Comment:
don't really want to keep all of this, but I will refactor the form to form-kit soon, so doesn't really matter